internal/keycloakb/configdbmodule.go

```diff
@@ -193,6 +213,43 @@ func (c *configurationDBModule) GetAuthorizations(ctx context.Context, realmID s
 	return res, nil
 }
 
+func (c *configurationDBModule) GetAuthorization(ctx context.Context, realmID string, groupName string, targetRealm string, targetGroupName string, actionReq string) error {
```

Comment:
It's not really GetAuthorization (suppose you are returning something)... AuthorizationExists? (AuthorizationExists would suppose you are returning a boolean)

Reply:
It is true, that this change do not match with the name. First I returned the authorization, and I made the changes you proposed but did not think to change the method name !